### PR TITLE
chore(clouddb): translations for clouddb

### PR DESCRIPTION
**Pull request by euhmeuh**
These are the translations for https://github.com/ovh-ux/ovh-manager-cloud/pull/1087
MANAGER-1379

### CODE REVIEW

File: client/app/vps/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -498,6 +498,42 @@
...
 
+   <translation id="vps_tab_cloud_database">Base de données</translation>
+   <translation id="vps_tab_cloud_database_description">Besoin de bases de données fiables, performantes et 100% managées par OVH ?<br/>Commandez notre service Cloud Databases !<br/>Vous disposerez d'une base de données autonome et compatible avec tous vos produits Cloud OVH.<br/>MariaDB, MongoDB, MySQL, PostgreSQL et Redis disponibles !</translation>
+   <translation id="vps_tab_cloud_database_header_name">Nom de l'instance</translation>
+   <translation id="vps_tab_cloud_database_header_version">Version de la base de donnée</translation>
+   <translation id="vps_tab_cloud_database_header_vps_authorized">VPS autorisé</translation>
+   <translation id="vps_tab_cloud_database_header_status">Statut</translation>
+   <translation id="vps_tab_cloud_database_table_empty">Aucune base de données trouvée.</translation>
+   <translation id="vps_tab_cloud_database_add_authorized_ip">Autoriser le VPS à accéder à cette base</translation>
+   <translation id="vps_tab_cloud_database_remove_authorized_ip">Retirer l'authorisation d'accéder à cette base</translation>
```
Comment (JeremyDec) [inline client/app/vps/translations/Messages_fr_FR.xml]: `authorisation` => `autorisation` in french

File: client/app/vps/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -498,6 +498,42 @@
...
+   <translation id="vps_tab_cloud_database_fetch_error">Une erreur est survenue lors du chargement des bases de données.</translation>
+   <translation id="vps_tab_cloud_database_whitelist_ip_name">Accès pour le VPS {{ vps }}</translation>
+   <translation id="vps_tab_cloud_database_whitelist_add_success">Le VPS a été authorisé par la base avec succès. Il peut maintenant accéder à celle-ci.</translation>
```
Comment (JeremyDec) [inline client/app/vps/translations/Messages_fr_FR.xml]: `authorisé` => `autorisé` (As above)

File: client/app/vps/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -498,6 +498,42 @@
...
+   <translation id="vps_tab_cloud_database_whitelist_add_error">Une erreur est survenue lors de la demande d'authorisation du VPS.</translation>
```
Comment (JeremyDec) [inline client/app/vps/translations/Messages_fr_FR.xml]: Same

File: client/app/vps/translations/Messages_fr_FR.xml (hunk 1 above)
Comment (euhmeuh) [reply client/app/vps/translations/Messages_fr_FR.xml]: :+1:
So much :gb: in me.

**Pull request merged**